js/src/lib/components/InferenceWidget/shared/WidgetInfo/WidgetInfo.svelte

```diff
@@ -6,36 +6,42 @@
 	export let model: WidgetProps["model"];
 	export let computeTime: string;
 	export let error: string;
-	export let modelLoadInfo: ModelLoadInfo = { status: "unknown" };
+	export let modelLoadInfo: ModelLoadInfo | undefined;
 
 	const status = {
-		error: "⚠️ This model could not be loaded by the inference API. ⚠️",
-		loaded: "This model is currently loaded and running on the Inference API.",
-		unknown: "This model can be loaded on the Inference API on-demand.",
+		TooBig: "⚠️ This model is too large to be loaded by the inference API. ⚠️",
+		Loaded: "This model is currently loaded and running on the Inference API.",
+		Loadable: "This model can be loaded on the Inference API on-demand.",
 	} as const;
 
 	const azureStatus = {
-		error: "⚠️ This model could not be loaded.",
-		loaded: "This model is loaded and running on AzureML Managed Endpoint",
-		unknown: "This model can be loaded loaded on AzureML Managed Endpoint",
+		TooBig: "⚠️ This model is too large to be loaded by the inference API.",
+		Loaded: "This model is loaded and running on AzureML Managed Endpoint",
+		Loadable: "This model can be loaded loaded on AzureML Managed Endpoint",
```

Comment:
are those the only possible states in the `api-inference` codebase?

Reply:
https://github.com/huggingface/api-inference/blob/1351b5c75e410fe3d5adfc315d50be4df4ee2a6d/master-rs/src/resources/status.rs#L75